code/modules/projectiles/guns/throw/crossbow.dm

```diff
@@ -29,41 +21,48 @@
 	if(cell && severity)
 		emp_act(severity)
 
-/obj/item/gun/throw/crossbow/update_icon()
-	if(!tension)
-		if(!to_launch)
-			icon_state = "[initial(icon_state)]"
-		else
-			icon_state = "[initial(icon_state)]-nocked"
-	else
-		icon_state = "[initial(icon_state)]-drawn"
+obj/item/gun/throw/crossbow/update_icon()
+	icon_state = "[initial(icon_state)]"
+	if(cell)
+		icon_state += "-Cell"
+	if(to_launch)
+		icon_state += "-Loaded"
+	if(tension)
+		icon_state += "-Charged"
+	if(cell.charge >= 500 && tension)
+		icon_state +="-Hot"
 
 /obj/item/gun/throw/crossbow/examine(mob/user)
 	. = ..()
 	if(cell)
 		. += "<span class='notice'>\A [cell] is mounted onto [src]. Battery cell charge: [cell.charge]/[cell.maxcharge]"
+		update_icon()
 	else
 		. += "<span class='notice'>It has an empty mount for a battery cell.</span>"
 
-/obj/item/gun/throw/crossbow/modify_projectile(obj/item/I, on_chamber = 0)
-	if(cell && on_chamber && istype(I, /obj/item/arrow/rod))
+/obj/item/gun/throw/crossbow/modify_projectile(obj/I,)
+	if( tension && (cell.charge >= 1999))
 		var/obj/item/arrow/rod/R = I
 		visible_message("<span class='danger'>[R] plinks and crackles as it begins to glow red-hot.</span>")
-		R.throwforce = 15
+		R.throwforce = 40
```

Comment:
Seems a bit high... 40 damage is quite a bit of damage!

Reply:
Double the original, maybe 25, would be a more reasonable buff, but maybe 15 isn't bad either